client/src/containers/SearchPage/index.spec.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SearchPage from './index';
+import * as utils from '../../utils';
+
+utils.sortMoviesComporator = jest.fn().mockReturnValue([]);
```

Comment:
you can mock the whole module instead:
`jest.mock('../../utils', () => ({
  sortMoviesComparator: jest.fn(() => []),
 filterMoviesComporator : ...
}))`